webapp/app/help.html

```diff
@@ -546,6 +546,35 @@
         </div>
     </div>
 </div>
+<div class="col-xs-12">
+    <div class="panel panel-default">
+            <div class="panel-heading"><h3>URL Query String Parameters</h3></div>
+            <div class="panel-body panel-spacing">
+                <table id="shortcuts">
+                    <tr>
+                      <td><span class="kbd">nojobs</span></td>
```

Comment:
I think we could omit the `kbd` class here since they aren't key shortcuts, and maybe use plain text for the 'nojobs' or '&nojobs' or whatever makes sense for the query param.